ISO8601.opam

```diff
@@ -8,15 +8,27 @@ dev-repo: "git+https://github.com/ocaml-community/ISO8601.ml.git"
 doc: "http://ocaml-community.github.io/ISO8601.ml/"
 bug-reports: "https://github.com/ocaml-community/ISO8601.ml/issues"
 synopsis: "ISO 8601 and RFC 3999 date parsing for OCaml"
-build: [
-  ["dune" "build" "-p" name]
-  ["dune" "build" "@doc" "-p" name] {with-doc}
-  #["dune" "runtest" "-p" name] {with-test}
-]
 depends: [
-  "dune" {>= "1.0"}
+  "dune" {>= "2.7"}
   "base-unix"
-  "ocaml" {>= "3.12.1" }
+  "ocaml" {>= "4.05.0" }
```

Comment:
I think that iso8601 should still support 4.03, what does it need from a recent OCaml anyway?

Reply:
It was needed for ocb but I managed to make it work with 4.03.